README.md

```diff
@@ -13,13 +13,13 @@ cd morphfits
 poetry install
 ```
 
-**This step is mandatory.** Download the GALFIT binary corresponding to your system [here](https://users.obs.carnegiescience.edu/peng/work/galfit/galfit.html), and move it to the location of your choosing (e.g., your data directory, if it exists).
+**This step is mandatory for GALFIT.** Download the GALFIT binary corresponding to your system [here](https://users.obs.carnegiescience.edu/peng/work/galfit/galfit.html), and move it to the location of your choosing (e.g., your data directory, if it exists).
```

Comment:
Resolved by 238931f

Reply:
Provide a specific path as a recommendation

Reply:
i.e., make it very explicit in a command:

> For instance, it can live in `morphfits_root/galfit`

(The tree is too verbose to make it clear what needs to be done)